capstone-project/src/main/angular-webapp/src/app/info-window/info-window.component.html

```diff
@@ -13,13 +20,18 @@ <h1>{{animal}}</h1>
         <input #inputDescription placeholder="Description"><br>
         <div *ngIf="!user">
             <input #inputReporter placeholder="Reporter's Name"><br>
-            <button (click)="submit(inputAnimal.value, inputDescription.value, inputReporter.value)">Submit</button>
+            <form enctype="multipart/form-data">
+                <input #inputImage type="file" name="image" accept="image/png, image/jpg" (change)="postFile($event.target.files)"><br>
+            </form>
+            <button [disabled]="isUploading" (click)="submit(inputAnimal.value, inputDescription.value, inputReporter.value)">Submit</button>
         </div>
         <div *ngIf="user">
             <input #inputReporterUser value="{{user.name}}"><br>
-            <button (click)="submit(inputAnimal.value, inputDescription.value, inputReporterUser.value)">Submit</button>
+            <form enctype="multipart/form-data">
+                <input #inputImage type="file" name="image" accept="image/png, image/jpg" (change)="postFile($event.target.files)"><br>
+            </form>
+            <button [disabled]="isUploading" (click)="submit(inputAnimal.value, inputDescription.value, inputReporterUser.value)">Submit</button>
         </div>
```

Comment:
As far as I can tell, the only difference here is in the input element. 

Can we just do ngIf on that?